src/style.css

```
@@ -1 +1,116 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+body {
+  box-sizing: border-box;
+}
+
+form {
```

```suggestion
.search {
```
style by classes instead by tags. exception can be just body or html tags or some reset styles.